src/Span/StackSpanManager.php

```diff
@@ -16,6 +16,7 @@ class StackSpanManager implements SpanManagerInterface
     public function __construct()
     {
         $this->stack = new \SplStack();
+        $this->context = null;
```

Comment:
Please add phpdoc `@var SpanContext|null` over property, to make it clear which values are expected to be present in property `$context`